### PR TITLE
feat(cli): make policyID optional for policy create

### PR DESCRIPTION
**Pull request by hazedav**
## Summary
We want to discourage users from defining their own policyId. Also, I learned that the policyId's format is invalid.

## How did you test this change?
Automated integration tests added

## Issue
https://lacework.atlassian.net/browse/ALLY-835

### CODE REVIEW

File: integration/policy_test.go [hunk 1]
```diff
@@ -236,18 +238,18 @@ func TestPolicyCreateStdin(t *testing.T) {
...
 	}
+	assert.Contains(t, out.String(), "was created.")
+	assert.Empty(t, stderr.String(), "STDERR should be empty")
+	assert.Equal(t, 0, exitcode, "EXITCODE is not the expected one")
 
 	policyID, err := getPolicyIdFromStdout(out.String())
-	assert.Nil(t, err)
+	if err != nil {
+		assert.FailNow(t, err.Error())
+	}
```
Comment (afiune) [inline integration/policy_test.go:248]: why these changes?

**Review by afiune — APPROVED** [review]
tenor-228301063

File: integration/policy_test.go (hunk 1 above)
Comment (hazedav) [reply integration/policy_test.go:248]: We want to fail fast when unable to extract policyID.  Subsequent calls require a valid policyID and will implicitly fail w/o them....
